src/main/java/net/minecraftforge/client/settings/KeyModifier.java

```diff
@@ -158,7 +194,13 @@ public static KeyModifier valueFromString(String stringValue)
 
     public abstract boolean matches(int keyCode);
 
+    /**
+     * @deprecated use {@link #isActive(IKeyConflictContext)}
+     */
+    @Deprecated
     public abstract boolean isActive();
 
+    public abstract boolean isActive(@Nullable IKeyConflictContext conflictContext);
```

Comment:
Keeping this abstract and calling it does not 'fix' the binary issues, you need to have a default body for this that bounces to the other version.